packages/amplify-app/src/scripts/amplify-push.js

```diff
@@ -24,14 +25,25 @@ function getNamedProfiles() {
   return namedProfiles;
 }
 
-function checkIfProfileExists(profileToUse) {
+async function checkIfProfileExists(profileToUse) {
```

Comment:
functin name should be changed. Better yet split this into 2 functions. One for checking and other for asking the user which profile they want to use if selected profile is missing in the computer

Reply:
Separating into two functions